ci/check-params-env.sh

```
@@ -679,14 +810,14 @@ function check_image_variable_matches_name_and_commitref_and_size() {
     abs_percent_change=${percent_change#-}
     test ${abs_percent_change} -le ${SIZE_PERCENTUAL_TRESHOLD} || {
         echo "Image size changed by ${abs_percent_change}% (expected: ${expected_img_size} MB; actual: ${actual_img_size} MB; treshold: ${SIZE_PERCENTUAL_TRESHOLD}%)."
-        return 1
+        return 2
     }
     # 2. Absolute size change
     size_difference=$((actual_img_size - expected_img_size))
     abs_size_difference=${size_difference#-}
     test ${abs_size_difference} -le ${SIZE_ABSOLUTE_TRESHOLD} || {
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)